src/components/posts/Post.js

```diff
@@ -5,7 +5,7 @@ import "./posts.css";
 import "../mainContent/mainContent.css";
 export default function Post(props) {
   return (
-    <div className="post-container content-container ">
+    <div className="post-container content-container " id={props.name}>
```

Comment:
Is the only function of the `props.name` prop to pass on this `id`? If so, I think it may be better to just manually set the `id` of the component when we make it, though I'm not sure what's best practice here.